Orange/widgets/cpewidget.py

```diff
@@ -79,38 +91,46 @@ def property_changed(self, name):
                 else:
                     openchar, closechar = "{", "}"
                 content = getattr(self, name)
-                # handle empty entry
-                if not content:
-                    content = "[]"
-                # handle missing brackets
-                if content[0] not in openchars:
-                    content = "[" + content
-                if content[-1] not in closechars:
-                    content = content + "]"
-                # strip whitespace between brackets and list
-                content = content[0] + content[1:-1].strip() + content[-1]
-                # replace semicolons by commas and warn
-                if ";" in content:
-                    content.replace(";", ",")
-                    warn_once("Semicolons are not a valid character in "
-                              "NeuroPype lists (replacing by commas).")
-                # handle missing commas in MATLAB-style list literals
-                if " " in content:
-                    if "'" in content:
-                        pass  # assume that these are quoted strings, ignore
-                    elif "," not in content:
-                        content = ", ".join(content.split())
-                # fix up specific bracket type
-                if content[0] in openchars and (content[0] != openchar):
-                    content = openchar + content[1:]
-                if content[-1] in closechars and (content[-1] != closechar):
-                    content = content[:-1] + closechar
+                # parse list default value and return
+                if content in NoneUIValue:
+                    content = None
```

Comment:
Hey Ale, much better now. I was actually suggesting to set the content to the string "None", then you don't need the special treatment in the try/except clause any more